src/index.js

```diff
@@ -3,33 +3,37 @@ var searchBtn = document.getElementById("search");
 var infoSkeleton = document.getElementById("infoSkeleton")
 var infoWord = document.getElementById("infoWord")
 var definition = document.getElementById("definition")
-var example = document.getElementById("example")
+ var example = document.getElementById("example")
 var synonym = document.getElementById("synonym")
 var dataDisplay = document.getElementById("data_display")
 var phoneticPara = document.getElementById("phoneticPara")
 var toggleBtn = document.getElementById("toggleBtn")
 var body = document.getElementById("body")
+var error = document.getElementById("error")
+var audio = document.getElementById("audio")
+var audioBtn = document.getElementById("audioBtn")
+var phoneticPara = document.getElementById("phoneticPara")
 
 
 
 
-            // Functions///
+            
     //API consumption
- async function fetchApi(word) {
-     infoSkeleton.classList.remove('hidden')
-     dataDisplay.style.display = 'none'
-    try {
-        let url = `https://api.dictionaryapi.dev/api/v2/entries/en/${word}`; 
-        const res = await fetch(url);
-        const data = await res.json();
-        infoSkeleton.classList.add('hidden')
-        dataDisplay.style.display = 'block'               
-            displayData(data, word);
-           
-        
-    } catch (error) {
-        console.log(error);
-    }}
+async function fetchApi(word) {
+    infoSkeleton.classList.remove('hidden')
+    dataDisplay.classList.add('hidden')
+    error.classList.add('hidden')
+   try {
+       let url = `https://api.dictionaryapi.dev/api/v2/entries/en/${word}`; 
+       const res = await fetch(url);
+       const data = await res.json();
+       infoSkeleton.classList.add('hidden')
+       data[0].word === 'undefined' ? dataDisplay.classList.add('hidden') : dataDisplay.classList.remove('hidden')     
```

Comment:
Double check this condition @Riri202. if you're checking whether `data[0].word` is not present, try `!data[0].word ? do this : do that`.

`undefined !== 'undefined'`

Reply:
I am trying to resolve 'undefined' being displayed on the page when it loads because that is what the api returns onload but I understand that the ternary operation I wrote means the user can't search the word 'undefined'. I don't know how to resolve it. 
![Screen Shot 2022-02-26 at 12 29 36](https://user-images.githubusercontent.com/92611151/155841362-1f129be1-3208-45d0-8c60-3f3aff3fe934.png)